src/main/java/org/folio/rest/impl/GetOrdersByIdHelper.java

```diff
@@ -0,0 +1,250 @@
+package org.folio.rest.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionException;
+
+import javax.ws.rs.core.Response;
+
+import org.apache.log4j.Logger;
+import org.folio.orders.rest.exceptions.HttpException;
+import org.folio.orders.utils.HelperUtils;
+import org.folio.rest.jaxrs.model.CompositePurchaseOrder;
+import org.folio.rest.jaxrs.model.PoLine;
+import org.folio.rest.jaxrs.resource.OrdersResource.GetOrdersByIdResponse;
+import org.folio.rest.tools.client.interfaces.HttpClientInterface;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Context;
+import io.vertx.core.Future;
+import io.vertx.core.Handler;
+import io.vertx.core.http.HttpMethod;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import me.escoffier.vertx.completablefuture.VertxCompletableFuture;
+
+public class GetOrdersByIdHelper {
+
+  private static final Logger logger = Logger.getLogger(GetOrdersByIdHelper.class);
+
+  public static final String BASE_MOCK_DATA_PATH = "mockdata/";
+
+  private final HttpClientInterface httpClient;
+  private final Context ctx;
+  private final Handler<AsyncResult<javax.ws.rs.core.Response>> asyncResultHandler;
+  private final Map<String, String> okapiHeaders;
+
+  public GetOrdersByIdHelper(HttpClientInterface httpClient, Map<String, String> okapiHeaders,
+      Handler<AsyncResult<Response>> asyncResultHandler, Context ctx) {
+    this.httpClient = httpClient;
+    this.okapiHeaders = okapiHeaders;
+    this.ctx = ctx;
+    this.asyncResultHandler = asyncResultHandler;
+  }
+
+  public CompletableFuture<CompositePurchaseOrder> getOrder(String id, String lang) {
+    CompletableFuture<CompositePurchaseOrder> future = new VertxCompletableFuture<>(ctx);
+
+    getCompositePurchaseOrderById(id, lang)
+      .thenAccept(orders -> {
+        logger.info("Returning mock data: " + JsonObject.mapFrom(orders).encodePrettily());
+        future.complete(orders);
+      })
+      .exceptionally(t -> {
+        logger.error("Error getting orders", t);
```

Comment:
probably trivial  but wouldn't it be easier to debug if we specify the error message that  we are trying to get order by id

Reply:
You get that for free if using the default log4j.properties:

`16:29:56 ERROR GetOrdersByIdHelper  Error getting orders
java.util.concurrent.CompletionException: org.folio.orders.rest.exceptions.HttpException: non-existent-po-id`